airsenal/scripts/fill_transfersuggestion_table.py

```diff
@@ -146,6 +147,7 @@ def optimize(
             strat_dict["chips_played"] = {}
             new_squad = squad
             gw = gameweek_range[0] - 1
+            strat_dict["root_gw"] = gw
```

Comment:
Is this the right gameweek to take as "root_gw" ?   I did some quick tests and it looks like the discount factor for 1 week ahead is already being applied to the next gameweek - could that be because of this minus one here?

Reply:
yeah I think you're correct.